.env.lmel

```diff
@@ -6,3 +6,4 @@ PROFILES_ASSETS_PATH=https://lememe.fr
 ACCESSIBLE_CONTRIBUTORS_IDS="[25, 35, 40, 43, 61, 118, 130, 140, 153, 161, 165, 166, 167, 170, 172, 173, 174, 175, 176, 177, 178, 179, 181, 182, 183, 60]"
 PRESELECTED_CONTRIBUTORS_IDS="[25, 43, 166, 167, 174, 60]"
 POPULAR_CONTRIBUTORS_IDS="[53, 54, 59, 46, 65]"
+FAMOUS_CONTRIBUTORS_PROFILE_SIDEBAR=[53, 59, 54, 25, 65]
```

Comment:
53, 59, 54, and 65 are not in the accessible contributors in LMEL … @MaartenLMEM what should be the famous contributors in LMEL ?

Reply:
Let's say : 166, 174, 160, 153, 167 @JalilArfaoui